envs/scib-pipeline.yml

```diff
@@ -14,21 +14,21 @@ dependencies:
   - openblas
   - llvmlite
   - libgcc-ng
+  - numba<=0.46  # for mnnpy
+  - anndata2ri
   - r-base
   - r-essentials
   - r-devtools
   - r-stringi
   - bioconductor-scater
   - bioconductor-scran
   # Methods
-  - scvi==0.6.7
-  - scanorama==1.7.0
-  - bbknn==1.3.9
-  - r-seurat==3.1.1
-  - numba<=0.46  # for mnnpy
-  - anndata2ri==1.0.5  # 1.0.6 has issues with HDF5 conversion
+  - scvi=0.6.7
+  - scanorama=1.7.0
+  - bbknn=1.3.9
+  - r-seurat=3.1.1
   - pip:
-    - git+git://github.com/theislab/scib.git
+    - scib==1.0.0
```

Comment:
everywhere else only 1 equals sign?

Reply:
Yes, I think pip syntax is different from conda syntax